assets/css/structures/_definition-content.scss

```
@@ -62,6 +62,11 @@
       // padding-left: 1rem;
       grid-column: 2;
     }
+
+    & h3 ~ ul {
+      border-bottom: 0.1rem solid lightgrey;
```

```suggestion
      border-bottom: 0.1rem solid lightgrey;
```
Could we use one of our sass variable colours here so that the change will respect light-dark mode?